README.md

```diff
@@ -141,6 +141,19 @@ transforms=MakeTombstone
 transforms.MakeTombstone.type=io.aiven.kafka.connect.transforms.MakeTombstone
 ```
 
+### `FilterFields`
```

Comment:
Based on the scope and implementation, I'd reconsider the naming. Maybe `FilterByValueRegex`? (naming is hard)

Reply:
sure I will change it to FilterByValueRegex